pb_plugins/dcsdkgen/struct.py

```diff
@@ -5,42 +5,40 @@
                     is_request,
                     is_response,
                     is_struct)
-from .name import NameParser
+from .name_parser import NameParser
 from jinja2.exceptions import TemplateNotFound
 
 
 class Struct(object):
     """ Struct """
 
-    def __init__(self, package, template_env, pb_struct):
-        try:
-            self._template = template_env.get_template("struct.j2")
-        except TemplateNotFound:
-            # We don't always generate code related to structs (but still
-            # collect them)
-            pass
-
+    def __init__(self, plugin_name, package, template_env, pb_struct):
+        self._plugin_name = NameParser(plugin_name)
+        self._package = NameParser(package)
+        self._template = template_env.get_template("struct.j2")
```

Comment:
Was it intended to remove the `TemplateNotFound` exception?

Reply:
Yes, actually I need the `struct` template in the backend, too :sweat_smile: